torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1755,6 +1849,34 @@ def simplify(self, expr: "sympy.Expr") -> "sympy.Expr":
             # divisions simplified away
             if new_pows.issubset(pows) and new_rationals.issubset(rationals):
                 expr = new_expr
+
+        if isinstance(expr, sympy.Rel):
+            lhs_const = get_const(expr.lhs)
+            rhs_const = get_const(expr.rhs)
+            if (
+                    lhs_const is not None
+                    and rhs_const is not None
+                    and lhs_const != 0
+            ):
+                expr = type(expr)(expr.lhs - lhs_const, expr.rhs - lhs_const)  # type: ignore
+        if isinstance(expr, sympy.Eq) and isinstance(expr.lhs, FloorDiv):
```

Comment:
Should we always do this simplification even if the rhs is a symbol?

Reply:
It helps us get rid of `FloorDiv`. But, probably we shouldn't do this if there are some simplification rules that depend on that operation. I will move this out.